muesli/tests/lectureTests.py

```diff
@@ -43,7 +43,7 @@ def test_lecture_do_allocation(self):
         res = self.testapp.get('/lecture/do_allocation/%s' % 123456, status=404)
 
     def test_lecture_add_exam(self):
-        res = self.testapp.get('/lecture/add_exam/%s' % 123456, status=404)
+        res = self.testapp.get('/lecture/add_exam/exam/%s' % 123456, status=404)
```

Comment:
One new feature was to change how the creation of new exams and assignments is done. The category is chosen by clicking the corresponding link on the edit-exam page instead of from a dropdown menu. Because of this the category is encoded in the URL, for example /lecture/add_exam/exam or /lecture/add_exam/assignment will add an exam or an assignment. The URLs in the tests had to be changed accordingly.